### ISSUE TITLE
Getting error about module loading mysqlnd.so

### ISSUE THREAD

**Issue by maZahaca** [hardware/postfixadmin#7]
[20-Feb-2017 15:48:24] NOTICE: PHP message: PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/lib/php7.1/modules/mysqlnd.so' - Error relocating /usr/lib/php7.1/modules/mysqlnd.so: BIO_new_mem_buf: symbol not found in Unknown on line 0

Started this with docker-compose.yml:
```yml
postfixadmin:
  image: hardware/postfixadmin
  container_name: postfixadmin
  domainname: domain.ltd
  hostname: mail
  restart: always
  environment:
    - DBPASS=xxxxxxxxxx
  links:
    - mailserver
    - mariadb
```

**Comment by hardware** [comment]
@Wonderfall any idea ?

In the meantime, i rollback to php 7.0.16

**Comment by Wonderfall** [comment]
No idea, I've tried before and it worked well. I can see however this error with `php7.1-fpm -m`, but it doesn't seem to block anything since it's using `mysqli` driver (and not mysqlnd).

**Comment by hardware** [comment]
mysqli internally use mysql native driver. Despite this warning, postfixadmin works without any problem. I keep php 7.0.16 for the moment.

**Comment by maZahaca** [comment]
Hi
did you do something to remove the warning?

**Comment by hardware** [comment]
i rollback to php 7.0.16

**Comment by maZahaca** [comment]
ok, thanks

**Comment by Wonderfall** [comment]
@hardware This should be fixed in the php 7.1 packages.

**Comment by hardware** [comment]
Fixed.
